src/Symfony/Bundle/SecurityBundle/Security/Http/RememberMe/RememberMeServicesResolver.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SecurityBundle\Security\Http\RememberMe;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Security\Http\RememberMe\RememberMeServicesInterface;
+use Symfony\Component\Security\Http\RememberMe\RememberMeServicesResolverInterface;
+
+class RememberMeServicesResolver implements RememberMeServicesResolverInterface
+{
+    /**
+     * @var \Symfony\Component\DependencyInjection\ContainerInterface
+     */
+    private $serviceContainer;
+
+    /**
+     * @param ContainerInterface $serviceContainer
+     */
+    public function __construct(ContainerInterface $serviceContainer)
```

Comment:
You should consider the @stof's suggestion, but if for any reason the whole container is required here, it could be better to use a setter and extend ```Symfony\Component\DependencyInjection\ContainerAware```.